### DISCUSSION TITLE
How do I do a backlash in a specified string?

### DISCUSSION THREAD

**Discussion by olifurz** [CosmosOS/Cosmos#2076] [Q&A]
How do I do a backslash (\) in a specified string as seen in the picture below? I get the following errors:

"Newline in constant" under "
") and ; expected" after ;

![image](https://user-images.githubusercontent.com/97028115/150843517-80da0ff0-9421-4252-9595-ec460114079f.png)

**Comment by asterd-og** [comment]
You can do
```cs
Console.WriteLine(@"\");
```
or
```cs
Console.WriteLine("\\");
```

**Comment by ghost** [comment]
> How do I do a backslash (\) in a specified string as seen in the picture below? I get the following errors:
> 

Consider learning a programming language before using cosmos xd
